src/components/shared/basic/ReleaseProcess/TechnicalIntegration/index.tsx

```diff
@@ -99,6 +100,39 @@ export default function TechnicalIntegration() {
   const [techUserProfiles, setTechUserProfiles] = useState<string[]>([])
   const [enableUserProfilesErrorMessage, setEnableUserProfilesErrorMessage] =
     useState(false)
+  const [selectedEncoding, setSelectedEncoding] = useState<string>('utf-8')
+  const [uploadFileInfo, setUploadFileInfo] = useState<DropzoneFile[]>([])
+  const unicodeSelectItems = [
+    {
+      id: 1,
+      title: 'utf-8',
+      value: 'utf-8',
```

Comment:
Hey, doesn't it understand uppercase 'UTF-8'? - I think it should also work and looks nicer if all are upper case